server/server.ts

```diff
@@ -1,58 +1,72 @@
 import path from "path";
 import express from "express";
-import sslRedirect from 'heroku-ssl-redirect';
+import sslRedirect from "heroku-ssl-redirect";
 import mongoose from "mongoose";
 import { MongoMemoryServer } from "mongodb-memory-server";
 import cors from "cors";
 import dotenv from "dotenv";
-import { fetchAddCourses } from "./dbInit";
+import { fetchAddCourses } from "./db/dbInit";
 import { configure } from "./endpoints";
 
 dotenv.config();
 const app = express();
-app.use(sslRedirect([
-  'development',
-  'production',
-]));
+app.use(sslRedirect(["development", "production"]));
 app.use(cors());
 app.use(express.static(path.join(__dirname, "../../client/build")));
 
 function setup() {
   const port = process.env.PORT || 8080;
-  app.get('*', (_, response) => response.sendFile(path.join(__dirname, '../../client/build/index.html')));
+  app.get("*", (_, response) => response.sendFile(path.join(__dirname, "../../client/build/index.html")));
```

Comment:
Is our app server-side rendering? Do we only run this `server.ts` script and we serve the index page here -> and then the rest of it is just handled with react-router? No script to run/serve the frontend separately? 

Not 100% sure how our deployment works yet.